<launch>

<rosparam file="$(find inmoov_servobrain)/config/config.yaml"/>

<group ns="servobus/00" >
  <node pkg="rosserial_python" type="serial_node.py" name="bus" respawn="true">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="115200"/>
  </node>
</group>

<node pkg="inmoov_servobrain" type="servobrain_loader.py" name="servobrain_loader" respawn="false"/>

<node pkg="inmoov_bringup" type="joint_command_dispatcher.py" name="joint_command_dispatcher" respawn="true"/>

<node pkg="inmoov_bringup" type="joint_status_dispatcher.py" name="joint_status_dispatcher" respawn="true"/>

<node pkg="inmoov_bringup" type="smartservo_status_dispatcher.py" name="motor_status_dispatcher" respawn="true"/>

<node pkg="inmoov_servobrain" type="trainer.py" name="servobrain_trainer" respawn="true"/>


<!--node pkg="inmoov_bringup" type="enable_manager.py" name="enable_manager" respawn="true"/-->

<!--node pkg="inmoov_bringup" type="rviz_manager.py" name="rviz_manager" respawn="true"/-->


</launch>